### PR TITLE
Update specification

### PR DESCRIPTION
**Pull request by sporejack**
TBA

### CODE REVIEW

File: specification.md [hunk 1]
```diff
@@ -62,15 +53,15 @@ The `OrderSide` type is an enumerated type with the following fields:
...
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| ID | 256-bit unsigned integer | The unique identifier of the order |
+| ID | 32-byte Keccak-256 digest | The unique identifier of the order |
 | Address | Ethereum address | The Ethereum address of the trader submitting this order |
-| Market | Ethereum address | The Ethereum address of the Tracer market |
+| Market | Ethereum address | The Ethereum address of the Tracer Perpetual Swaps market |
 | Side | `OrderSide` | The side of the market for the order |
 | Price | 256-bit unsigned integer | The price of the order |
 | Amount | 256-bit unsigned integer | The quantity of the order |
```
Comment (raymogg) [inline specification.md:67]: order creation field is needed. Maybe an example json input of this in the `submit` function too just to clarify field names (eg no capitals)

File: specification.md [hunk 2]
```diff
@@ -316,105 +355,33 @@ An example request payload is:
...
+    "user": "0xD62bf94a99c87dc7B3924A880020B6B316F536E6",
+    "target_tracer": "0x61f5a3d36c10d7ea1d797e70f7486eb7ad177481",
     "side": "Ask",
-    "price": 4380090000,
-    "amount": 4,
-    "expiration": 1595997399,
+    "price": "4380090000",
+    "amount": "4000000000",
+    "expiration": "1895997399",
     "signedData": "0xcafebeefdeaddeadcafebeefdeaddeadcafebeefdeaddeadcafebeefdeaddeadcafebeefdeaddeadcafebeefdeaddeadcafebeefdeaddeadcafebeefdeaddeadff"
```
Comment (raymogg) [inline specification.md:366]: created field

**Review by raymogg — CHANGES_REQUESTED** [review]

File: specification.md (hunk 2 above)
Comment (sporejack) [reply specification.md:366]: Consult 9e3b09f.

File: specification.md (hunk 1 above)
Comment (sporejack) [reply specification.md:67]: Consult 9e3b09f.

**Review by raymogg — APPROVED** [review]

**Pull request merged**